src/pages/createBookmark.js

```diff
@@ -0,0 +1,39 @@
+import { bookmarkView } from "../views/index.js";
+
+export const createBookmark = (event) => {
+  console.log("create bookmark", event.target.id);
+  const movies = JSON.parse(localStorage.getItem("moviesList")) || [];
+  const selectedMovie = [];
+  movies.filter((movie) => {
+    if (Number(movie.id) === Number(event.target.value)) {
+      selectedMovie.push(movie);
+    }
+    return movie.id === event.target.value;
+  });
```

Comment:
I already commented on this in a previous review. Please check the documentation again on how `.map()` and `.filter()` should be used. Both methods return a new array. If you are not using that returned array then you are doing something wrong.